### PR TITLE
feat(pages): created white and g100 example pages

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6732

### Description
Created the White and Gray 100 page examples.

### Changelog

**New**

- created new test pages for Carbon themes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,11 @@
     "@actions/core": "^1.2.6",
     "@carbon/ibmdotcom-react": "^1.24.0",
     "@carbon/ibmdotcom-styles": "^1.24.0",
+    "@carbon/ibmdotcom-web-components": "1.10.0-canary.1243270330.0",
```
Comment (jeffchew) [inline package.json:32]: @IgnacioBecerra Would you be able to use the rc version for this now?

**Comment by jeffchew** [comment]
Getting this for some reason when viewing the page 
Screenshot_20210921-065055

File: styles/g100-theme-example.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+@import "~@carbon/ibmdotcom-styles/scss/components/callout-quote/index";
+@import "~@carbon/ibmdotcom-styles/scss/components/card-link/card-link";
+@import "~@carbon/ibmdotcom-styles/scss/components/content-block-cards/content-block-cards";
+@import "~@carbon/ibmdotcom-styles/scss/components/content-block-media/index";
+@import "~@carbon/ibmdotcom-styles/scss/components/content-block-segmented/index";
+@import "~@carbon/ibmdotcom-styles/scss/components/content-group-horizontal/content-group-horizontal";
+@import "~@carbon/ibmdotcom-styles/scss/components/content-group-cards/index";
+@import "~@carbon/ibmdotcom-styles/scss/components/content-group-pictograms/index";
+@import "~@carbon/ibmdotcom-styles/scss/components/ctasection/index";
+@import "@carbon/ibmdotcom-styles/scss/components/feature-card/feature-card";
```
Comment (kennylam) [inline styles/g100-theme-example.scss:15]:
```suggestion
@import "~@carbon/ibmdotcom-styles/scss/components/feature-card/feature-card";
```

**Review by kennylam — COMMENTED** [review]
@IgnacioBecerra `g100` theme has a few components looking a bit off. Fixing the `feature-card` import might help resolve.

**Comment by jeffchew** [comment]
Looking at the image files, am thinking we should probably avoid a mix of upper caase and lower case in the file names. Should probably stick with all lower case.

**Comment by IgnacioBecerra** [comment]
@kennylam Hadn't noticed the `~` thing, thanks! As for the other components in the dark page, I'm actually confused since everything shows up just fine locally. 
https://user-images.githubusercontent.com/24970122/134425916-0b487154-5ba8-4f40-a3b5-d5ce2a2805ff.mov

Though sometimes after constant reloading it does go back to how it looks like in the preview. I'm wondering if this might have to do with the way the styles are being loaded in NextJS? Seems like the default/"white" styles override the dark theme colors, so the text ends up blending with the background.

@jeffchew Now using lowercase for the files and the current rc.1 - had that specific version since canary at the time hadn't updated with a React wrapper that I needed.

**Pull request merged**